core/slim_sim.cpp

```diff
@@ -71,7 +71,8 @@ extern "C" {
 static const char *SLIM_TREES_FILE_VERSION_INITIAL __attribute__((unused)) = "0.1";		// SLiM 3.0, before the Inidividual table, etc.; UNSUPPORTED
 static const char *SLIM_TREES_FILE_VERSION_PRENUC = "0.2";		// before introduction of nucleotides
 static const char *SLIM_TREES_FILE_VERSION_POSTNUC = "0.3";		// SLiM 3.3.x, with the added nucleotide field in MutationMetadataRec
-static const char *SLIM_TREES_FILE_VERSION = "0.4";				// SLiM 3.4.x onward, with the new model_hash key in provenance
+static const char *SLIM_TREES_FILE_VERSION = "0.4";				// SLiM 3.4.x, with the new model_hash key in provenance
+static const char *SLIM_TREES_FILE_VERSION_NEW = "0.5";				// SLiM 3.x.x onward, with information in metadata instead of provenance
```

Comment:
it will be 3.5.0

Reply:
This should be 3.5.x not 3.x.x

Reply:
I changed this, you're looking at an outdated version.

Reply:
aargh